webpack/assets/javascripts/react_app/components/PF4/TypeAhead/TypeAhead.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react';
+import Downshift from 'downshift';
```

Comment:
would be nice to see a comment here to refresh my memory on what `Downshift` does for us